sormas-backend/src/main/java/de/symeda/sormas/backend/event/EventService.java

```diff
@@ -736,33 +732,25 @@ private Predicate createEventDateFilter(CriteriaQuery<?> cq, CriteriaBuilder cb,
 		Date eventDateTo = eventCriteria.getEventDateTo();
 
 		if (eventDateType == null || eventDateType == EventCriteriaDateType.EVENT_DATE) {
+			Predicate eventDateFilter = null;
```

Comment:
I think the event filters aren't correct yet; they basically need to do the same as the quarantine filters (because both event start and end date are optional, such as the quarantine start and end date); also, report date must only be used as a fallback in case NEITHER event start or end date are specified; if I'm not mistaken, the current logic will always fall back to the report date even if the event date is specified.